src/main/64D0.c

```diff
@@ -1,4 +1,5 @@
 #include "common.h"
```

Comment:
main.h already includes common.h, so the include for common.h here needs to go :P